klusterletconfig/v1alpha1/types.go

```diff
@@ -72,6 +72,11 @@ type KlusterletConfigSpec struct {
 
 	// InstallMode is the mode to install the klusterlet
 	InstallMode *InstallMode `json:"installMode,omitempty"`
+
+	// +optional
+	// BootstrapKubeConfigSecrets is a list of secrets that reflects the Klusterlet.Spec.RegistrationConfiguration.BootstrapKubeConfigs.
+	// Currently, it only supports the LocalSecrets type of Klusterlet BootstrapKubeConfigs.
+	BootstrapKubeConfigSecrets []corev1.ObjectReference `json:"bootstrapKubeConfigSecrets,omitempty"`
```

Comment:
this is different from what we can define in klusterlet, why not do a 1-1 mapping？

Reply:
There are 2 reasons:

#1

The api defined in ocm for the purpose of future flexibility, the struct is deep: BootstrapKubeConfigs.LocalSecrets.KubeConfigSecrets

But in import-controller we only have one secret type.

#2

On hub side, the bootstrapkubeconfigs can be set per cluster, which mean the defination in ocm is not suite because it doesn't have the namespace(manged cluster namespace):
```yaml
type KubeConfigSecret struct {
	// Name is the name of the secret.
	// +required
	Name string `json:"name"`
}
```

Reply:
You're right about this, the API should be 1:1 mapping to ocm api. I've update the PR. Please take another review.